tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -29,7 +29,7 @@ Feature: full text search
       | spaces           |
 
 
-  Scenario Outline: sharee searches a file tagged by sharer using tag
+  Scenario Outline: sharee searches shared files using a tag
```

Comment:
@saw-jan what is the best practice while writing a scenario?
Scenario: Scenario describes the steps and expected outcome for a particular test case